DOCUMENTATION.md

```diff
@@ -1089,108 +874,71 @@ By default, JSON serialization is used if the object is not `IdentifiedDataSeria
-are converted to JSON strings and transmitted to the Hazelcast members as such.
+If the Hazelcast Node.js client cannot find a suitable serializer for an object, it uses `JSON Serialization` by default. With `JSON Serialization`, objects are converted to JSON strings and transmitted to the Hazelcast members as such.
 
-When the Hazelcast Node.js client retrieves a JSON serialized data from a member, it parses the JSON string and returns the object represented by that
-string to the user. However, you may want to defer the string parsing and work with the raw JSON strings.
+When the Hazelcast Node.js client retrieves a JSON serialized data from a member, it parses the JSON string and returns the object represented by that string to the user. However, you may want to defer the string parsing and work with the raw JSON strings.
 
 For this purpose, you can configure your client to return `HazelcastJsonValue` objects when it retrieves a JSON serialized data from a member.
 
-`HazelcastJsonValue` is a lightweight wrapper around the JSON strings. You may get the JSON string representation of the object using the `toString` method.
+`HazelcastJsonValue` is a lightweight wrapper around the raw JSON strings. You may get the JSON string representation of the object using the `toString()` method.
 
 Below is the configuration required to return `HazelcastJsonValue` objects instead of JavaScript objects.
 
 **Programmatic Configuration:**
 
 ```javascript
-config.serializationConfig.jsonStringDeserializationPolicy = JsonStringDeserializationPolicy.NO_DESERIALIZATION;
-```
-
-**Declarative Configuration:**
-
-```json
-{
-    "serialization": {
-        "jsonStringDeserializationPolicy": "no_deserialization"
+const cfg = {
+    serialization: {
+        jsonStringDeserializationPolicy = 'NO_DESERIALIZATION'
     }
-}
+};
 ```
 
-# 5. Setting Up Client Network
 
-All network related configuration of Hazelcast Node.js client is performed via the `network` element in the declarative configuration file, or in the object `ClientNetworkConfig` when using programmatic configuration. Let's first give the examples for these two approaches. Then we will look at its sub-elements and attributes.
-
-**Declarative Configuration:**
-
-Here is an example of configuring the network for Node.js Client declaratively.
-
-```json
-{
-    "network": {
-        "clusterMembers": [
-            "10.1.1.21",
-            "10.1.1.22:5703"
-        ],
-        "smartRouting": true,
-        "redoOperation": true,
-        "connectionTimeout": 6000
-    }
-}
-```
+# 5. Setting Up Client Network
 
-**Programmatic Configuration:**
+All network related configuration of Hazelcast Node.js client is performed via the `network` element in the configuration. Let's first give the examples for these two approaches. Then we will look at its sub-elements and attributes.
```

Comment:
This may be a bit misleading right now since we have `connectionStrategy` element now.

Also, for the second sentence, we don't have two ways anymore.

Reply:
Nice catch. Changed to `Main parts of network related configuration for Hazelcast Node.js client is performed via the `network` configuration option.` in ab63f5f90cffccfd14871b3d175821732ebd2783